File: test7Go/oop/models/burger/simpleBurger.go
package burger

import (
	oop "oop/models"
)

type SimpleBurger struct {
	Filling string
	oop.CommandItem
}

func NewSimpleBurger(name string, price float64) SimpleBurger {
	filling := "bread, sauce, tomato, onion"
	commandItem := oop.CommandItem{Name: name, Price: price, Quantity: 1}

	return SimpleBurger{filling, commandItem}
}

File: test7C#/test7/Model/Burger/SimpleBurger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model
{
    class SimpleBurger: CommandItem
    {
        private string filling;

        public SimpleBurger(string name, Decimal price): base(name, price, 1)
        {
            this.filling = "bread, sauce, tomato, onion";
        }

        public string Filling()
        {
            return filling;
        }

        public void Filling(string filling)
        {
            this.filling = filling;
        }
    }
}
